phpunit/functional/RuleDefineItemtypeTest.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2025 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace tests\units;
+
+use DbTestCase;
+use PHPUnit\Framework\Attributes\DataProvider;
+
+/* Test for inc/ruleimportcomputer.class.php */
+
+class RuleDefineItemtypeTest extends DbTestCase
+{
+    /**
+     * Adds a new rule
+     *
+     * @param string $name     New rule name
+     * @param array  $criteria Rule criteria
+     * @param array  $action   Rule action
+     *
+     * @return int
+     */
+    protected function addRule(string $name, array $criteria, array $action): int
```

Comment:
There is already a `RuleBuilder` to do this kind of operations. Here is an example: https://github.com/glpi-project/glpi/blob/11ebd5ad4f42aaa35956f120fc34ed16addcfd69/phpunit/functional/SLMTest.php#L1444-L1450

Reply:
Changed to be used in new tests, and also from `RuleImportAssetTest`